extensions/ipynb/src/helpers.ts

```diff
@@ -108,8 +109,12 @@ function convertJupyterOutputToBuffer(mime: string, value: unknown): NotebookCel
 		} else if (mime.startsWith('image/') && typeof value === 'string' && mime !== 'image/svg+xml') {
 			// Images in Jupyter are stored in base64 encoded format.
 			// VS Code expects bytes when rendering images.
-			const data = Uint8Array.from(atob(value), c => c.charCodeAt(0));
-			return new NotebookCellOutputItem(data, mime);
+			if (typeof Buffer !== 'undefined' && typeof Buffer.from === 'function') {
```

Comment:
Fixes the issue where previously this part of the code would fail in desktop (node)